src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -376,7 +381,7 @@ class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave M
         //build a variant context of the HVCF with the hashes
         return createHVCFRecord(metaDataRecord.sampleName, Position(metaDataRecord.refContig,metaDataRecord.refStart),
             Position(metaDataRecord.refContig, metaDataRecord.refEnd ),
-            Pair(metaDataRecord.refSeq, assemblyHaplotypeHash))
+            Pair(metaDataRecord.refSeq[0].toString(), assemblyHaplotypeHash))
```

Comment:
Will `refSeq` always be a collection or non-nullable?

Reply:
refSeq is actually a string and I am getting the first allele here.  I will add in a error check to make sure there is something there so we won't have any errors pop up.